src/transformers/integrations.py

```diff
@@ -288,6 +298,49 @@ def dynamic_modules_import_trainable(*args, **kwargs):
     return best_run
 
 
+def run_hp_search_sigopt(trainer, n_trials: int, direction: str, **kwargs) -> BestRun:
+
+    from sigopt import Connection
+
+    conn = Connection()
+    proxies = kwargs.pop("proxies", None)
+    if proxies is not None:
+        conn.set_proxies(proxies)
+
+    experiment = conn.experiments().create(
+        name="huggingface-tune",
+        parameters=trainer.hp_space(None),
+        metrics=[
+            dict(name="objective", objective=direction, strategy="optimize"),
+        ],
```

Comment:
```suggestion
        metrics=[dict(name="objective", objective=direction, strategy="optimize")],
```
This fits in one line

Reply:
fixed.